content/posts/newsletter-february-2024.md

```diff
@@ -0,0 +1,32 @@
+title: Read the Docs Newsletter - February 2024
+date: Jan 1, 2024
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community, build-in-public
+authors: Eric Holscher
+status: published
+
+## News and updates
+
+* We moved our new blog posts to be published on our website, so going forward you can find it [directly on our website](https://about.readthedocs.org/blog/).
+* We shipped an update that allows [Sphinx builds to share environments](https://github.com/readthedocs/readthedocs.org/pull/11073). This should make builds of multiple formats faster.
```

Comment:
Shouldn't we use sentence break here?

Reply:
Sentence breaks are hard to do in list items with indention, so not super worried about it.